src/server/Services/StoreUpdateNotifierService.cs

```diff
@@ -15,15 +15,17 @@ public class StoreUpdateNotifierService : IDisposable
     private Timer? _timer;
     private bool disposedValue;
     private readonly string _cacheDirectory;
```

Comment:
`_cacheDirectory` is no longer set but is used throughout.

Reply:
That can be removed